modules/@apostrophecms/page/index.js

```diff
@@ -77,8 +77,7 @@ module.exports = {
       // array of pages.
 
       async getAll(req) {
-        // Edit access to draft is sufficient to fetch either
-        self.publicApiCheck(req, 'draft');
+        self.publicApiCheck(req);
```

Comment:
Didn't you just recently do the opposite?

Reply:
No, before it was checking whether you could edit, now it's enough just to be able to view drafts in general on the site, which is a better test that doesn't run afoul of six reasons why it's not permissive enough.

Reply:
Because of this change I no longer need to pass a mode override, view-draft is itself a permission.